x-pack/legacy/plugins/uptime/public/components/functional/location_map/embeddables/map_config.ts

```diff
@@ -12,8 +12,12 @@ import { LocationPoint } from './embedded_map';
  * destination, and line layer for each of the provided indexPatterns
  *
  */
-export const getLayerList = (upPoints: LocationPoint[], downPoints: LocationPoint[]) => {
-  return [getLowPolyLayer(), getDownPointsLayer(downPoints), getUpPointsLayer(upPoints)];
+export const getLayerList = (
+  upPoints: LocationPoint[],
+  downPoints: LocationPoint[],
+  { gray, danger }: { gray: string; danger: string }
```

Comment:
There's actually an `UptimeAppColors` type that we could use here instead of declaring the interface inline.

Reply:
I have taken care of this code change in my other related PR to this file. So i will merge this.